res/values-sv/strings.xml

```diff
@@ -11,45 +11,46 @@
     <string name="PrefGroupFeedback">Återkoppling</string>
     <string name="PrefTheme">Tema</string>
-    <string name="PrefThemeLight">Light</string>
-    <string name="PrefThemeDark">Dark</string>
+    <string name="PrefThemeLight">Ljust</string>
+    <string name="PrefThemeDark">Mörkt</string>
     <string name="PrefStartScreen">Hemskärm</string>
     <string name="PrefStories">Berättelser</string>
     <string name="PrefWaterCoolerStories">Berättelser</string>
-    <string name="PrefSavedItems">Sparade Föremål</string>
+    <string name="PrefSavedItems">Sparade föremål</string>
     <string name="PrefDuckMode">Sök</string>
-    <string name="PrefRecentSearch">Senaste Sökningar</string>
-    <string name="PrefRecordHistory">Spara Senaste</string>
-    <string name="PrefClearHistory">Rensa Senaste</string>
-    <string name="PrefRecordCookies">Spara Cookies</string>
-    <string name="PrefClearCookies">Rensa Cookies</string>
-    <string name="PrefClearBrowserCache">Rensa Webbläsarens Cache</string>
-    <string name="PrefClearCacheAndCookies">Rensa Cookies och Cache</string>
-    <string name="PrefSources">Ändra Källor</string>
-    <string name="PrefResetSources">Återställ Källor</string>
+    <string name="PrefRecentSearch">Senaste sökningar</string>
+    <string name="PrefRecordHistory">Spara senaste</string>
+    <string name="PrefClearHistory">Rensa senaste</string>
+    <string name="PrefRecordCookies">Spara cookies</string>
+    <string name="PrefClearCookies">Rensa cookies</string>
+    <string name="PrefClearBrowserCache">Rensa webbläsarens cache</string>
+    <string name="PrefClearCacheAndCookies">Rensa cookies och cache</string>
+    <string name="PrefSources">Ändra källor</string>
+    <string name="PrefResetSources">Återställ källor</string>
     <string name="PrefRegion">Region Boost</string>
+    <!-- Need context to above -->
     <string name="PrefDirectQuery">Search on Tap</string>
-    <string name="PrefExternalBrowser">Använd Extern Webbläsare</string>
+    <string name="PrefExternalBrowser">Använd extern webbläsare</string>
     <string name="PrefNever">Aldrig</string>
     <string name="PrefExternalBrowserExceptSearches">För icke-sök-resultat</string>
     <string name="PrefExternalBrowserAlways">Alltid</string>
-    <string name="PrefClearCacheCookiesEveryLaunch">Varje Start</string>
+    <string name="PrefClearCacheCookiesEveryLaunch">Varje start</string>
     <string name="PrefClearCacheCookiesWeekly">Veckolig</string>
     <string name="PrefClearCacheCookiesMonthly">Månatlig</string>
-    <string name="PrefSendFeedback">Skicka Återkoppling</string>
-    <string name="PrefRate">Betygsätt App</string>
-    <string name="PrefAppSearch">Inkludera Apps</string>
+    <string name="PrefSendFeedback">Skicka feedback</string>
+    <string name="PrefRate">Betygsätt app</string>
+    <string name="PrefAppSearch">Inkludera appar</string>
     <string name="PrefMainFontSize">Fontstorlek</string>
```

Comment:
Please could this be changed to `Storlek på typsnittet` to be consistent with line 107?